packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -400,12 +412,6 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
     const azureAccount: AzureAccount =
       vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account")!.exports;
     AzureAccountManager.currentStatus = azureAccount.status;
-    if (AzureAccountManager.currentStatus === "LoggedIn") {
```

Comment:
If we call `updateSubscriptionInfo` both in this position and `runBackgroundAsyncTasks`, is there a problem?

Reply:
I think there's no problem, in this position it probably does nothing. I'll add it back, thanks.